README.quickstart.md

```diff
@@ -1,255 +0,0 @@
-
-The Quickstart configuration of Aryn automatically runs an example workload that downloads [this PDF](http://sortbenchmark.org/2004_Nsort_Minutesort.pdf) from the [Sort Benchmark website](http://www.sortbenchmark.org), uses [Sycamore](https://github.com/aryn-ai/sycamore) to prepare the data, and loads it into the Aryn platform. This gives you a out-of-the-box example for conversational search.
-
-You can then choose to crawl the rest of the [Sort Benchmark website](http://www.sortbenchmark.org) and load this data into your index. Instructions are [here](#Add-the-full-Sort-Benchmark-Dataset). Then, you can search over the whole dataset.
-
-Finally, [see below](#add-your-own-data) for instructions on how to crawl and load data from any website using this Sycamore processing script.
-
-## Deploying Aryn Search
-
-### Quickstart prerequisites
-
-1. An OpenAI Key for LLM access. You can create an OpenAI account [here](https://platform.openai.com/signup), or if you already have one, you can retrieve your key [here](https://platform.openai.com/account/api-keys).
-
-2. For the highest quality table extraction (and better answers), the demo Sycamore script needs AWS credentials for Amazon Textract and an Amazon S3 bucket for Textract input/output. This script uses Textract and Amazon S3 in the US-East-1 region. You can optionally disable Textract. You will accrue AWS charges for Textract usage. If you want to enable Textract:
-
-- If you do not have an AWS account, sign up [here](https://portal.aws.amazon.com/billing/signup). You will need this during configuration.
-- Create an Amazon S3 bucket in your AWS account in the US-East-1 region for use with Textract (e.g. s3://username-textract-bucket). We recommend you set up bucket lifecycle rules that automatically delete files in this bucket, as the data stored here is only needed temporarily during a Sycamore data processing job.
-
-### Now, let's get started
-
-1. Download the Docker compose files from the Quickstart repo. You will need the [compose](https://github.com/aryn-ai/quickstart/blob/main/compose.yaml) and [.env](https://github.com/aryn-ai/quickstart/blob/main/.env) files.
-
-```
-git clone git@github.com:aryn-ai/quickstart.git
-```
-
-2. Set up your Aryn Search environment:
-
-```
-export SYCAMORE_TEXTRACT_PREFIX=s3://your-bucket-name-here
-export OPENAI_API_KEY=YOUR-KEY
-```
-
-Textract is used by default with the demo Sycamore script, and you can choose to enable it or disable it.
-
-- To use it, you need to configure your AWS credentials. You can enable AWS SSO login with [these instructions](https://docs.aws.amazon.com/cli/latest/userguide/sso-configure-profile-token.html#sso-configure-profile-token-auto-sso), or you can use other methods to set up AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, and if needed AWS_SESSION_TOKEN.
-
-If using AWS SSO:
-
-```
-aws sso login --profile YOUR-PROFILE-NAME
-eval "$(aws configure export-credentials --format env --profile YOUR-PROFILE-NAME)"
-```
```

Comment:
I still refer to this. Where can I find it outside of this doc?

Reply:
docs/source/welcome_to_sycamore/get_started.md